outputs.tf

```diff
@@ -15,56 +15,56 @@
  */
 
 output "network_name" {
-  value       = google_compute_network.network.name
+  value       = module.vpc.network_name
   description = "The name of the VPC being created"
 }
 
 output "network_self_link" {
-  value       = google_compute_network.network.self_link
+  value       = module.vpc.network_self_link
   description = "The URI of the VPC being created"
 }
 
 output "svpc_host_project_id" {
```

Comment:
Let's just call this `project_id`.

Reply:
Do you still want it only outputted if `shared_vpc_host == true` ? 

The original logic was `svpc_host_project_id` would only be outputted if you enabled shared vpc, otherwise it was `""`.
But outputting just `project_id` seems like you'd still expect a value even with shared vpc off

Reply:
Yes, let's output it in both cases.